Syntaxes/Chapel.tmLanguage

```diff
@@ -269,7 +273,7 @@
 		<dict>
 			<key>match</key>
 			<string>(?x)\b(
-              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln
+              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln | writebits | writef | readbits | readf
```

Comment:
I think it's better to leave this change for #22 and not include it in this pr.

Reply:
Yeah got pushed in the same branch , anyways deleting that #22

Reply:
I don't see an update to remove that change, what's the status on this?

Reply:
Yeah, I got a little busy with assignments for few days, fixing this today only . Sorry for the delay.